app/helpers/podcasts_helper.rb

```diff
@@ -101,4 +101,58 @@ def disable_non_compliant_language(language)
       language
     end
   end
+
+  def subscribe_link_platform_label(link)
+    I18n.t("helpers.label.podcast.subscribe_link.#{link.platform}")
+  end
+
+  def subscribe_link_id_label(link)
```

Comment:
difference between this and the above method is that this is used for the field label, and the above is used as the label next to the icon and in the dropdown.